jscomp/others/jsxDOM.ml

```diff
@@ -42,26 +39,31 @@ type domProps = {
   (* accessibility *)
   (* https://www.w3.org/TR/wai-aria-1.1/ *)
   (* https://accessibilityresources.org/<aria-tag> is a great resource for these *)
-  (* [@bs.optional] [@bs.as "aria-current"] ariaCurrent: page|step|location|date|time|true|false, *)
+  ariaCurrent: [`page | `step | `location | `date | `time | `_true | `_false]
```

Comment:
Nice to have the missing aria attributes here, but the
```
`_true
```

will just be `'_true'` in JS, won't it? So this is not really correct.

Reply:
Ah! Yes, it is. I have to find another way.

Reply:
Maybe comment it out again for now.
Can be revisited when this is converted to res later on.

Reply:
```ocaml
type domProps<'ariaCurrent, ... > = { ... ; ariaCurrent: 'ariaCurrent; ... }
```
```rescript
// ReactDOM.res
type domProps<'ariaCurrent, ... > = JsxDOM.domProps = { ... , ariaCurrent: 'ariaCurrent, ... }

type ariaCurrent = [#"true" | #"false"]

@variadic @module("react")
external createDOMElementVariadic: (
  string,
  ~props: domProps<ariaCurrent, ...>=?,
  array<React.element>,
) => React.element = "createElement"
```
It can be one of temporary solution.

Reply:
https://github.com/rescript-lang/rescript-compiler/pull/5706/commits/a6b136d75aef61306236d339f76a237d0e081df7 and https://github.com/rescript-lang/rescript-react/pull/63/commits/23389412e3918eb23b3f7b87cdac1aacd3b5bcbe

Reply:
The thing is, it would be difficult to change later on, as coordination is needed between compiler and rescript-react.

Reply:
Right, it makes sense. I’ll revert it.

Reply:
commented `aria*` and `loading` fields which are inevitable to express with the underscore.